pkg/registry/resolver.go

```diff
@@ -59,9 +59,13 @@ func (r *registry) Resolve(name string, config api.MultiStageTestConfiguration)
 				config.Environment[k] = v
 			}
 		}
+		if config.AllowSkipOnSuccess == nil {
```

Comment:
Why only when is this not `nil`?

Reply:
Because if you define it at this level you want to override what was set in the "workflow" that you're reusing. At least that's my understanding. You can be re-using a workflow that has some settings but you want to be able to override them. At least in this case.

Reply:
If you don't define it directly in the multi-step test you'll get what is in the workflow.